src/Cake.Common/Tools/DotNetCore/Restore/DotNetCoreRestoreSettings.cs

```diff
@@ -14,6 +14,11 @@ public sealed class DotNetCoreRestoreSettings : DotNetCoreSettings
         public string Source { get; set; }
```

Comment:
@adamhathcock please in this PR remove `string Source { get; set; }` and it's usages in favor of the new `ICollection<string> Sources { get; set; }`

We've discussed internally with in @cake-build/team even though it's a breaking change, to keep API clean and consistent, and as we feel we catch this early.